numbers/guides/hostedMessaging.md

```diff
@@ -157,6 +180,23 @@ Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
 
 ```
 
+{% sample lang="php" %}
+
+```php
+print_r($account->checkTnsPortability(array("5554443333", "5553334444"))->ImportTnCheckerPayload);
```

Comment:
Can you break this out to an example more like the ruby where the response is stored as a variable

Reply:
done 👍